documentation/staging/content/release-notes.md

```diff
@@ -129,7 +129,7 @@ draft: false
   * Added `.spec.failureRetryIntervalSeconds` and `.spec.failureRetryLimitMinutes` for configuration of failure retry timing.
   * Added `.spec.serverPod.maxReadyWaitTimeSeconds` for configuring maximum time before considering servers as stalled during startup.
 
-* Condition status updates.
+* [Condition status]({{< relref "/managing-domains/accessing-the-domain/status-conditions.md" >}}) updates.
```

Comment:
As we have no other links in this particular section, I recommend removing this link. Instead, use it as replacement for the local link at line 76 under 'major themes' section.